### ISSUE TITLE
Pills in the taglist component are not accessiblie

### ISSUE THREAD

**Issue by vsgoulart** [bpmn-io/form-js#413]
__Describe the Bug__

When using that taglist component if a user selects a value the pills of the selected values in the taglist component are not accessible.

![image](https://user-images.githubusercontent.com/9433996/203269785-204eb0ea-a0ce-4a89-a855-fb7cf74b9ccd.png)

__Steps to Reproduce__

1. Create a form with a taglist and render it with the form viewer
2. Select a value in the taglist
3. Now with keyboard only try to remove a value
4. Selected values can't be removed with keyboard only (no outline and pills are not clickable/focusable)

__Expected Behavior__

Values should be able to be removed with keyboard only and the pills should be focusable with an outline

__Environment__

- Host (Browser/Node version), if applicable: [e.g. MS Edge 18, Chrome 69, Node 10 LTS]
- OS: [e.g. Windows 7]
- Library version: [e.g. 2.0.0]

**Comment by pinussilvestrus** [comment]
I believe we should have a look at this rather soon.
